recipe/meta.yaml

```diff
@@ -50,10 +52,12 @@ test:
 
 about:
   home: http://www.openssl.org/
-  license: Apache-style
-  license_family: Apache
   license_file: LICENSE
+  license: Apache
```

Comment:
Not actually Apache hence the `-style` above. In fact, we should probably just say `OpenSSL`.